CHANGELOG.markdown

```diff
@@ -1,3 +1,10 @@
+naxt major
+----------
+
+* Add `Eq1`, `Ord1`, `Read1` and `Show1` instances for `Coyoneda` and `Yoneda`.
+* Change contexts of `Eq` and `Ord` instances of `Coyoneda` and `Yoneda` to require
```

Comment:
Is there a reason you changed the `Eq` and `Ord` instances, but not the `Read` and `Show` instances?